src/pseudopeople/configuration/interface.py

```diff
@@ -0,0 +1,51 @@
+"""
+===========================
+The Configuration Interface
+===========================
+
+An interface for users to interact with the configuration (config) in pseudopeople.
+
+In ``pseudopeople`` a configuration is used to provide noise level that get passed to the noise functions to apply
+various types of noise to given datasets.  Users can provide their own configuration that will override the default
+values.
+.. code-block:: python
```

Comment:
Is this actually rendering as a code block? I can't get it to work on my end

Reply:
It is now.